guides/source/configuring.md

```diff
@@ -2329,6 +2329,10 @@ If set to `true`, cookies will be written even if this criteria is not met.
 
 This defaults to `true` in `development`, and `false` in all other environments.
 
+#### `config.action_dispatch.verbose_redirect_logs`
+
+Specifies if source locations of redirects should be logged below relevant log lines. By default, the flag is `true` in development and `false` in all other environments.
```

Comment:
Sorry, by reading this, and I already have an application,  I understand that in development I don't have to do anything or set anything in `development.rb`,
But by reading the PR, I need to enable it manually in `config/environments/development.rb` 

Maybe something like `By default in new applications, the flag is `true`` would make more sense.

Reply:
It should be turned on automatically in development. Sorry, I'll look into this too.